client/app/components/sections/Topbar/Topbar.css

```diff
@@ -52,10 +54,19 @@
     font-size: var(--Menu_fontSizeSmall);
   }
 
+  & > * {
+    margin-right: 24px;
+  }
+
+}
+
+.topbarSpacer {
+  flex-grow: 100;
```

Comment:
It's to create the spacing between the menu and language dropdown and push the items to left and right sides, like the design for desktop shows. `flex-grow` seems to be a bit ambiguous number, didn't find any definition for that but `30` seemed to be good value